CHANGELOG.md

```diff
@@ -8,6 +8,105 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+| [Portal](https://github.com/eclipse-tractusx/portal) | portal: [2.1.0-RC2](https://github.com/eclipse-tractusx/portal/releases/tag/portal-2.1.0-RC2) | [2.1.0-RC2](https://github.com/eclipse-tractusx/portal/releases/tag/portal-2.1.0-RC2) |
+| [Portal - portal-assets](https://github.com/eclipse-tractusx/portal-assets) | n/a | [2.0.0](https://github.com/eclipse-tractusx/portal-assets/releases/tag/v2.0.0) |
+| [Portal - portal-backend](https://github.com/eclipse-tractusx/portal-backend) | n/a | [2.1.0-RC1](https://github.com/eclipse-tractusx/portal-backend/releases/tag/v2.1.0-RC1) |
+| [Portal - portal-frontend](https://github.com/eclipse-tractusx/portal-frontend) | n/a | [2.1.0-RC2](https://github.com/eclipse-tractusx/portal-frontend/releases/tag/v2.1.0-RC2) |
+| [Portal - portal-frontend-registration](https://github.com/eclipse-tractusx/portal-frontend-registration) | n/a | [2.0.1-RC2](https://github.com/eclipse-tractusx/portal-frontend-registration/releases/tag/v2.0.1-RC2) |
+| [PURIS](https://github.com/eclipse-tractusx/puris) | puris: [2.6.3](https://github.com/eclipse-tractusx/puris/releases/tag/puris-2.6.3) | [2.0.2](https://github.com/eclipse-tractusx/puris/releases/tag/2.0.2) |
+| [PURIS Kit](https://eclipse-tractusx.github.io/docs-kits/next/category/puris-kit) | n/a | [0.3.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/PURIS%20Kit/PURIS%20Kit%20Changelog) |
+| [Self Description (SD) Factory](https://github.com/eclipse-tractusx/sd-factory) | sdfactory: [2.1.21](https://github.com/eclipse-tractusx/sd-factory/releases/tag/sdfactory-2.1.21) | [2.1.13](https://github.com/eclipse-tractusx/sd-factory/releases/tag/v2.1.13) |
+| [Semantic Hub](https://github.com/eclipse-tractusx/sldt-semantic-hub) | semantic-hub: [0.3.1](https://github.com/eclipse-tractusx/sldt-semantic-hub/releases/tag/semantic-hub-0.3.1) | [0.4.0](https://github.com/eclipse-tractusx/sldt-semantic-hub/releases/tag/v0.4.0) |
+| [SSI Credential Issuer](https://github.com/eclipse-tractusx/ssi-credential-issuer) | ssi-credential-issuer: [1.1.0-rc.1](https://github.com/eclipse-tractusx/ssi-credential-issuer/releases/tag/ssi-credential-issuer-1.1.0-rc.1) | [1.1.0-rc.1](https://github.com/eclipse-tractusx/ssi-credential-issuer/releases/tag/v1.1.0-rc.1) |
+| [Trace-X Traceability Application](https://github.com/eclipse-tractusx/traceability-foss) | helm-charts: [1.3.42](https://github.com/eclipse-tractusx/traceability-foss/releases/tag/helm-charts-1.3.42) | [13.0.0](https://github.com/eclipse-tractusx/traceability-foss/releases/tag/13.0.0) |
+| [Traceability Kit](https://eclipse-tractusx.github.io/docs-kits/next/category/traceability-kit) | n/a | [5.2.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/Traceability%20Kit/Traceability%20Kit%20Changelog) |
```

Comment:
Add Supply Chain Disruption Notifications KIT (links to be checked in the end):
```suggestion
| [Traceability Kit](https://eclipse-tractusx.github.io/docs-kits/next/category/traceability-kit) | n/a | [5.2.0](https://eclipse-tractusx.github.io/docs-kits/next/kits/Traceability%20Kit/Traceability%20Kit%20Changelog) |
| [Supply Chain Disruption Notifications Kit](https://eclipse-tractusx.github.io/docs-kits/next/category/supply-chain-disruption-notifications-kit) | n/a | [1.0.0](https://eclipse-tractusx.github.io/docs-kits/kits/Supply%20Chain%20Disruption%20Notifications%20KIT/page_changelog.md) |
```
see https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/987

Reply:
@tom-rm-meyer-ISST is this a new KIT? Cause it is already in the added section. thx

Reply:
@stephanbcbauer Ah sorry, seems like I missed to search for it in the added section. You're right, resolved.